articles/synapse-analytics/backuprestore/restore-sql-pool.md

```diff
@@ -138,7 +138,7 @@ Steps:
 
 1. Update Az.Sql Module to 3.8.0 (or greater) if on an older version using `Update-Module`. Otherwise it will cause failures. To validate your version via PowerShell:
    ```powershell
-   foreach ($i in (get-module -ListAvailable | ?{$_.name -eq 'az.sql'}).Version) { $version = [string]$i.Major + "." + [string]$i.Minor; if ($version -gt 3.7) {write-host "Az.Sql version $version installed. Prequisite met."} else {update-module az.sql} }
+   foreach ($i in (get-module -ListAvailable | ?{$_.name -eq 'az.sql'}).Version) { $version = [string]$i.Major + "." + [string]$i.Minor; if ($version -gt 3.7) {write-host "Az.Sql version $version installed. Perquisite met."} else {update-module az.sql} }
```

Comment:
This is not a good word, I guess idea was to use "Prerequisite" and not "Perquisite"

Reply:
Opps. good catch ! Thank you !